docs/docs/accessibility.md

```diff
@@ -0,0 +1,277 @@
+internal page anchors and some styling:
+
+- [WebAIM - Skip Navigation Links](http://webaim.org/techniques/skipnav/)
+
+Also use landmark elements and roles, such as `<main>` and `<aside>`, to demarcate page regions as assistive technology allow the user to quickly navigate to these sections.
+
+Read more about the use of these elements to enhance accessibility here:
+
+- [Deque University - HTML 5 and ARIA Landmarks](https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html)
+
+### Programmatically managing focus
+
+Our React applications continuously modify the HTML DOM during runtime, sometimes leading to keyboard focus being lost or set to an unexpected element. In order to repair this, 
+we need to programmatically nudge the keyboard focus in the right direction. For example by resetting keyboard focus to a button that opened a modal window after that modal window is closed.
+
+The Mozilla Developer Network takes a look at this and describes how we can build [keyboard-navigable JavaScript widgets](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets).
+
+To set focus in React, we can use [Refs to Components](refs-and-the-dom.html) or [ReactDOM](react-dom.html).
+
+Using refs we can first create a ref to an element in the JSX of a component class:
+
+```javascript{2-3,7}
+render() {
+  // Use the `ref` callback to store a reference to the text input DOM
+  // element in an instance field (for example, this.textInput).
+  return (
+    <input
+      type="text"
+      ref={(input) => { this.textInput = input; }} />
+  );
+}
+```
+
+Then we can focus it elsewhere in our component when needed:
+
+ ```javascript
+ focus() {
+   // Explicitly focus the text input using the raw DOM API
+   this.textInput.focus();
+ }
+ ```
+
+Or we could use ReactDOM to directly find and focus the element without a ref:
+
+```javascript
+focus() {
+  //Get a handle to the DOM node for our component.
+  const componentNode = ReactDOM.findDOMNode(this);
+  //Find the required element and explicitly focus the text input 
+  //using the raw DOM API
+  componentNode.querySelector('input').focus();
+}
+```
+
+A great focus management example is the [react-aria-modal](https://github.com/davidtheclark/react-aria-modal). This is a relatively rare example of a fully accessible modal window. Not only does it set initial focus on 
+the cancel button (preventing the keyboard user from accidentally activating the success action) and trap keyboard focus inside the modal, it also resets focus back to the element that 
+initially triggered the modal.
+
+**Note:** While this is a very important accessibility feature, it is also a technique that should be used judiciously. Use it to repair the keyboard focus flow when it is disturbed, not to try and anticipate how
```

Comment:
We have existing styling format for notes:

```
> Note:
>
> (note text)
```

Let's use it here too.
The "Note:" will be bolded automatically.